src/TestSuite/Fixture/ChecksumTestFixture.php

```diff
@@ -113,11 +99,13 @@ protected function _hash(ConnectionInterface $db): string
 
         if ($driver instanceof Mysql) {
             $sth = $db->execute("CHECKSUM TABLE " . $this->table . ';');
+
```

Comment:
these newlines seem to be unrelated to compat with 4.x

Reply:
I got this error in phpcs:

Missing blank line before return statement